Documentation/AdditionalFunctionality/Jobs.md

```diff
@@ -0,0 +1,132 @@
+![document type: vision](https://apodini.github.io/resources/markdown-labels/document_type_vision.svg)
+
+# Jobs
+
+`Job`s allow to schedule repeating tasks and are in contrast to `Component`s not request based.
+Use cases for `Job`s could be running background database operations or fetching data from a remote server.
+In Apodini such tasks are defined by conforming to the `Job` protocol which includes a `run` method.
+This method is executed at scheduled points in time or when triggered by an [ObservableObject](<./../Communicational\ Patterns/2.\ Tooling/2.4.\ ObservedObject.md>).
+
+## Usage
+
+`Job`s can include the `@Environment` property wrappers to use other services in the `run` method.
+The following example defines a `Job` and uses the `@Environment` property wrapper to inject the `NotificationCenter` in order to send out push notifications.
+
+```swift
+struct MondayService: CronJob {
+    @Environment(\.notificationCenter) var notificationCenter: NotificationCenter
+
+    func run() {
+        notificationCenter.send(alert: .init(title: "It's monday. The beginning of a new week"), to: "newsletter")
+    }
+}
+```
+
+## Scheduling
+
+Apodini `Job`s can be scheduled using the common syntax from [crontab](https://man7.org/linux/man-pages/man5/crontab.5.html).
+
+```
+┌───────────── minute (0 - 59)
+│  ┌───────────── hour (0 - 23)
+│  │  ┌───────────── day of the month (1 - 31)
+│  │  │  ┌───────────── month (1 - 12)
+│  │  │  │  ┌───────────── day of the week (0 - 6)
+│  │  │  │  │              (Sunday to Saturday)
+│  │  │  │  │
+│  │  │  │  │
+\* \* \* \* \*
+```
+
+`Job`s are scheduled at server startup from the `configuration` property using the `Schedule` configuration and the cron expression.
+When scheduling `Job`s, Apodini will create a single instance which can be used with `@ObservedObject` in other components.
```

Comment:
Maybe even with an `@Environment` in addition to `@ObservedObject`. It might just be that a `Handler` just wants to great an `@Published` property of a `Job` but doesn't need to be triggered on an update of a `Job`.